TestsUserCreationOperation.cs

```diff
@@ -0,0 +1,174 @@
+﻿using Newtonsoft.Json;
+using NUnit.Framework;
+using RestSharp;
+
+namespace APIAutomation.Tests
+{
+    [TestFixture]
+    public class UserCreationTests
+    {
+        private RestClient _clientForReadScope;
+        private RestRequest _requestForReadScopeGetZipCodes;
+        private RestRequest _requestForReadScopeGetUsers;
+
+        private RestClient _clientForWriteScope;
+        private RestRequest _requestForWriteScope;
+
+        [SetUp]
+        public void Setup()
+        {
+            var clientForReadScope = ClientForReadScope.GetInstance("http://localhost:8000", "0oa157tvtugfFXEhU4x7", "X7eBCXqlFC7x-mjxG5H91IRv_Bqe1oq7ZwXNA8aq");
+            _clientForReadScope = clientForReadScope.GetRestClient();
+            _requestForReadScopeGetZipCodes = new RestRequest("/zip-codes", Method.Get);
```

Comment:
If I'm not mistaken, it's not nessesary to pass Get parameter for RestRequest constructor because it's already set there as default parameter.

Reply:
Ok, it works in such way